styles/_theme.scss

```diff
@@ -9,6 +9,10 @@
     background-color: $secondary;
   }
 
+  input[type="radio"]:checked + label {
+    font-weight: 700;
+  }
```

Comment:
The theme file should only be used for colors. Put input styles other than colors in the form file. If you do it like this, it only applies to radio labels inside a color class, which may not always be the case.